graphics/scaler.h

```diff
@@ -23,6 +23,7 @@
 #define GRAPHICS_SCALER_H
 
 #include "common/scummsys.h"
+#include "graphics/managed_surface.h"
```

Comment:
Please, avoid includes like this in the header files.

Instead, declare that class, e.g. `class ManagedSurface;` and include the header in the relevant C++ file.

In this header you are using only the pointer, thus, the class declaration is enough